zkvm/src/verifier.rs

```diff
@@ -79,10 +79,11 @@ impl<'a, 'b> Delegate<r1cs::Verifier<'a, 'b>> for Verifier<'a, 'b> {
 impl<'a, 'b> Verifier<'a, 'b> {
     /// Verifies the `Tx` object by executing the VM and returns the `VerifiedTx`.
     /// Returns an error if the program is malformed or any of the proofs are not valid.
-    pub fn verify_tx<'g>(tx: Tx, bp_gens: &'g BulletproofGens) -> Result<VerifiedTx, VMError> {
+    pub fn verify_tx(tx: Tx) -> Result<VerifiedTx, VMError> {
+        let bp_gens = BulletproofGens::new(128, 1);
```

Comment:
We cannot hardcode this for two reasons:

1. Bigger txs will have larger circuits and more than 128 multipliers, failing the verification if 128 is hardcoded.
2. Computing generators is expensive. Validator would want to precompute them and use over its whole lifetime to validate arbitrary amount of transactions, so we need to pass it in.